DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/README.md

```diff
@@ -0,0 +1,198 @@
+# Merge Sort
```

Comment:
Please have a look at the changes Audrey made here: https://github.com/oneapi-src/oneAPI-samples/pull/507/ and replicate those changes in your README